plugins/logging/pkg/gateway/drivers/backend/mock_driver.go

```diff
@@ -0,0 +1,89 @@
+package backend
```

Comment:
We've been putting drivers of this form in `pkg/test` : https://github.com/rancher/opni/blob/main/pkg/test/test_cluster_driver.go

Reply:
This doesn't use the same register pattern though so that won't work here due to cyclic dependencies.  It could probably work with some more refactoring though so I will create a tech debt card to come back to this.

Reply:
https://github.com/rancher/opni/issues/1280